frontend/src/components/Login.jsx

```diff
@@ -20,10 +23,17 @@ export default function Login() {
         "Content-Type": "application/json",
       },
       body: JSON.stringify(dataPost),
-    }).then((res) => {
-      console.warn(res);
-      navigate("/");
-    });
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.token) {
+          setToken(data.token);
+          navigate("/");
+        }
+      });
+    if (!token) {
+      setErrorInput(true);
+    }
```

Comment:
nice :)

just one issue : you're testing token outside the fetch, so because of asynchronism you're testing a state value before getting a response from your server. try:

    .then((data) => {
      if (data.token) {
        setToken(data.token);
        navigate("/");
      } else {
        setErrorInput(true);
      }